### ISSUE TITLE
hyperv-vmcx fails

### ISSUE THREAD

**Issue by johnsout** [hashicorp/packer#8407]
#### Overview of the Issue
Hi, I 've been trying to use my base VM for building a new Hyper-v machine without any luck in the last two days.  It keeps failing on Compare VM Null reference exception.

#### Reproduction Steps

```bash
packer build -force -var 'sserver=product/SQLstream-sServer-6.0.0-x64.run'  -var 'streamlab=product/Guavus-StreamLab-2.3.1.19048-linux-x64.run' -var 'release=6.0.0.1' -var 'rel=6.0.0' -var 'trim=6.0.0' vhd.json 
```

### Packer version

packer 1.4.5

### Simplified Packer Buildfile

```json
"builders":[
      {
  "type": "hyperv-vmcx",
  "clone_from_vmcx_path" : "C:\\Users\\aspen\\vhd",
  "vm_name": "SQLstream-{{user `release`}}",
  "ssh_username": "sqlstream",
  "ssh_password": "sqlstream",
  "switch_name":"{{user `hyperv_switchname`}}",	
  "communicator":"ssh",
  "http_directory":"http",
  "enable_secure_boot":"false",
	"generation" :"1",
	"guest_additions_mode":"disable",
  "headless":"{{user `headless`}}",
  "output_directory":"c:\\Users\\aspen\\SQLstream-{{user `release`}}",
  "shutdown_command":"sudo systemctl poweroff"
  }
],
```
### Operating system and Environment details

Windows Server 2012

### Log Fragments and crash.log files
https://gist.github.com/johnsout/02923b5db172fbf135911c041c24543a

**Comment by nywilken** [comment]
hi @johnsout thanks for opening up this issue. I updated the issue description to fix formatting and point directly to the debug log gist; we'll try to look at this when we get a chance

**Comment by johnsout** [comment]
Hi, did you have a chance to try this?

**Comment by SwampDragons** [comment]
Hi, I just noticed that you're on v1.4.5. I believe this issue was solved in the 1.5.0 release: https://github.com/hashicorp/packer/pull/8357

Try updating to the most recent version of Packer and let me know if the issue is resolved. thanks!

**Comment by SwampDragons** [comment]
Closing because we never heard back. Hopefully 1.5 solved this.
